content/4.examples/0.index.md

```diff
@@ -0,0 +1,72 @@
+---
+navigation: false
+---
+
+# Examples
+
+This section contains examples of how to use the various features of the Gevamu Payments Solution.
+
+## Architecture
+
+Abstract CorDapp will be used as a sample to demonstrate possible architecture of your app integrated with the Gevamu Payments Solution.
+
+```mermaid
+---
+title: Dependencies
+---
+
+flowchart LR
+  target[Your App]
+  net.corda:corda-core --> target
+  com.gevamu.corda:payments-workflows --> target
+  com.gevamu.corda:payments-contracts --> target
+  custom[... your dependencies] --> target
+```
+
+Sample dependencies configuration in `build.gradle.kts` file:
+
+```kotlin [build.gradle.kts]
+repositories {
+    mavenCentral()
+    // Repository with corda artifacts
+    maven("https://software.r3.com/artifactory/corda")
+}
+
+dependencies {
+    // Core corda dependencies
+    implementation("net.corda:corda-core:4.9.3")
+    // Download SDK https://gevamu.github.io/corda-payments-sdk/
```

Comment:
```
// Gevamu dependencies
```
We assume that they will be published on central repository. There is no need to mention download.

Reply:
Looks like this comment is misleading.

```suggestion
    // Gevamu dependencies
```